configs/tsimx6ul_ubuntu_23_04_minimal_defconfig

```diff
@@ -0,0 +1,23 @@
+CONFIG_DS_DISTRO_UBUNTU_23_04=y
+CONFIG_DS_PACKAGELIST="ubuntu_23.04_headless.txt"
+CONFIG_DS_KERNEL_PROVIDER_GIT_URL="https://github.com/embeddedTS/linux-lts.git"
+CONFIG_DS_KERNEL_PROVIDER_GIT_VERSION="v5.10.170.6-ts"
+CONFIG_DS_KERNEL_DEFCONFIG="tsimx6ul_defconfig"
```

Comment:
Since this is minimal, should this use `tsimx6ul_minimal_defconfig` for kernel?

Reply:
Yes, the packagelist should be minimal as well.  Committed those and I'll rebase so I can test with the proc fix & run more testing on a board.